lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -4,6 +4,11 @@
 
 module I18n::Tasks::Translators
   class DeeplTranslator < BaseTranslator
+    # max allowed texts per request
+    BATCH_SIZE = 50
```

Comment:
50 seems a bit small, an average Rails app has 1,500+ keys. Perhaps we can increase this?

Reply:
yes, i know, but it ran with multiple request without problems for 4.5k keys. 
The limit itself is bound to the deepl api restriction: ' Up to 50 texts can be sent for translation in one request.'